snapcraft.yaml

```diff
@@ -60,8 +60,14 @@ parts:
       - python3-requests
       - python3-yaml
     override-pull: |
-      craftctl set version="$(/bin/date +%Y%m%d)"
       craftctl default
+      # detect whether we are doing a fips build on LP
+      if git remote get-url origin | grep "fips"; then
+        craftctl set version="$(/bin/date +%Y%m%d)+fips"
+        echo "SNAP_FIPS_BUILD=1" > ./.fips-env
+      else
+        craftctl set version="$(/bin/date +%Y%m%d)"
```

Comment:
Not important. But I feel we should remove `.fips-env` in here. Or even make it empty. The only case I think of however is if one changes the "origin" remote. So I am being picky for not much.

I wonder, could someone just add that file manually to force a fips build? Maybe this is what you had in mind on how to build locally the fips build, unfortunately, it does not add the +fips in that case.

Reply:
I'll add a line to remove it in that case - but no, I've been testing by doing `git remote set-url origin ...` to rename it and then just rename it back